frontend/src/scenes/insights/trendsLogic.ts

```diff
@@ -153,11 +153,13 @@ export const trendsLogic = kea<trendsLogicType<FilterType, ActionType, TrendPeop
                         )
                         response = response.result
                     } else {
```

Comment:
Noticed this part of logic has no typing (results is typed as `any[]`) - worth adding it since you're here.

Reply:
removed changes here as they were unnecessary so will make these changes separately